src/Raven.Server/RavenServer.cs

```diff
@@ -1157,9 +1157,16 @@ private void ListenToNewTcpConnection(TcpListener listener)
                 }
                 catch (Exception e)
                 {
-                    if (_tcpLogger.IsInfoEnabled)
+                    if (_tcpLogger.IsOperationsEnabled)
```

Comment:
Move the log statement to the `if`

Reply:
Also, we need normal exception to be in info, and the retries in operations

Reply:
So the first time should be logged as `info` and the rest as `operations`?